GDJS/Runtime/pixi-renderers/pixi-filters-tools.js

```diff
@@ -1,5 +1,8 @@
 gdjs.PixiFiltersTools = function() {};
 
+const clampValue = (value, min, max) => Math.max(min, Math.min(max, value));
```

Comment:
We're in the game engine, so ES5 only ;)
`var clampValue = function(value, min, max) { return Math.max(min, Math.min(max, value)); }`

Also, remember that classical JavaScript is a pain when it comes to global variable and variable scope. Using var to declare a variable outside of any scope like this will create a global variable, both polluting the global namespace and risking our function to be erased by something else. We don't want to run this risk, so instead be sure to always declare things inside functions (in which case they are function scoped) or attach them as "static" members:

`gdjs.PixiFiltersTools.clampValue = function(value, min, max) { return Math.max(min, Math.min(max, value)); }`;

then use it as `filter.uniforms[parameterName] = gdjs.PixiFiltersTools.clampValue(value, 0, 1);`

More verbose, but that's the price of a nice "self contained" code that won't break even if someone mess with the global namespace.

Reply:
(note that the "someone mess with the global namespace" could be actually another extension declaring a `clampValue` with a different implementation - imagine the strange bug and how hard to debug it is "my game effects are broken when I use this unrelated extension")

Reply:
Wrote about this here: https://github.com/4ian/GDevelop/commit/5c648e3f2b8c444bd99221cea56b92c00cdddddd